posts/tests/test_api.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+
+from django.test import TestCase
+
+from posts.tests.test_views import ModelCreator
+
+
+class TestApi(TestCase):
+    def setUp(self):
+        self.creator = ModelCreator()
+        self.user = self.creator.create_user()
+        self.time = datetime.now()
+        self.expected_time_fmt = self.time.isoformat("T") + "Z"
+
+    def test_date_rfc3339(self):
+        post = self.creator.create_post(
+            is_visible=True,
+            is_public=True,
+        )
+        post.published_at = self.time
+        post.updated_at = self.time
+        converted_post = post.to_dict()
+        self.assertEqual(converted_post["date_published"], self.expected_time_fmt)
```

Comment:
А дату тогда так можно проверять:

```suggestion
        self.assertIsNotNone(
            datetime.strptime(converted_post["date_published"],  '%Y-%m-%dT%H:%M:%S%z').tzinfo,
        )
```